docs/core/diagnostics/cli-tools/dotnet-dump.md

```diff
@@ -0,0 +1,141 @@
+    dumpil <arguments>                   Displays the Microsoft intermediate language (MSIL) that is associated with a managed method.
+    dumplog <arguments>                  Writes the contents of an in-memory stress log to the specified file.
+    dumpmd <arguments>                   Displays information about a MethodDesc structure at the specified address.
+    dumpmodule <arguments>               Displays information about a EE module structure at the specified address.
+    dumpmt <arguments>                   Displays information about a method table at the specified address.
+    dumpobj <arguments>                  Displays info about an object at the specified address.
+    dso, dumpstackobjects <arguments>    Displays all managed objects found within the bounds of the current stack.
+    eeheap <arguments>                   Displays info about process memory consumed by internal runtime data structures.
+    finalizequeue <arguments>            Displays all objects registered for finalization.
+    gcroot <arguments>                   Displays info about references (or roots) to an object at the specified address.
+    gcwhere <arguments>                  Displays the location in the GC heap of the argument passed in.
+    ip2md <arguments>                    Displays the MethodDesc structure at the specified address in code that has been JIT-compiled.
+    name2ee <arguments>                  Displays the MethodTable structure and EEClass structure for the specified type or method in the specified module.
+    pe, printexception <arguments>       Displays and formats fields of any object derived from the Exception class at the specified address.
+    syncblk <arguments>                  Displays the SyncBlock holder info.
+    histclear <arguments>                Releases any resources used by the family of Hist commands.
+    histinit <arguments>                 Initializes the SOS structures from the stress log saved in the debuggee.
+    histobj <arguments>                  Examines all stress log relocation records and displays the chain of garbage collection relocations that may have led to the address passed in as an argument.
+    histobjfind <arguments>              Displays all the log entries that reference an object at the specified address.
+    histroot <arguments>                 Displays information related to both promotions and relocations of the specified root.
+    setsymbolserver <arguments>          Enables the symbol server support
+    soshelp <arguments>                  Displays all available commands when no parameter is specified, or displays detailed help information about the specified command. soshelp <command>
+```
+
+## Docker special instructions
+
+If you're running under docker, dump collection requires `SYS_PTRACE` docker capabilities (`--cap-add=SYS_PTRACE or --privileged`).
```

Comment:
Make sure my change is correct here. I wanted to get the or word out for localization.

```suggestion
If you're running under Docker, the dump collection requires `SYS_PTRACE` Docker capabilities (`--cap-add=SYS_PTRACE` or `--privileged`).
```